odoo/fields.py

```diff
@@ -2143,6 +2143,8 @@ def _setup_attrs(self, model, name):
             # We cannot use field.selection or field.selection_add here
             # because those attributes are overridden by ``_setup_attrs``.
             if 'selection' in field.args:
+                if self.related:
+                    _logger.warning("%s: selection attribute doesn't make sense because the field is related (in %s module)", self, self._module)
```

Comment:
Instead of "does not make sense" maybe something like "will be ignored". Also don't add the module in the message, the field should be enough.

Reply:
Done :+1: